lite-pm/pages/components/Garbage.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter } from "reactstrap";
+import styles from "../../styles/Garbage.module.css";
+import Task from "./Task";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+
+import uuid from "react-uuid";
+
+export default class Garbage extends Component {
+  constructor(props) {
+    super(props);
+    this.childRef = React.createRef();
+  }
+
+  render() {
+    return (
+      <div className={styles.container} ref={this.childRef}>   
+          <FontAwesomeIcon width={18} icon={faPlus} className="mr-2" />   
```

Comment:
Should this be a garbage icon?

Reply:
Yes, I will update that in my next merge with the colours